lib/autospec/qunit_runner.rb

```diff
@@ -5,17 +5,25 @@ module Autospec
   class QunitRunner < BaseRunner
 
     WATCHERS = {}
-    def self.watch(pattern, &blk); WATCHERS[pattern] = blk; end
-    def watchers; WATCHERS; end
+    def self.watch(pattern, &blk)
+      WATCHERS[pattern] = blk
+    end
```

Comment:
Can you add a newline here?

Reply:
So we should really enable `Layout/EmptyLineBetweenDefs` as well ;)

https://github.com/rubocop-hq/rubocop/blob/master/manual/cops_layout.md#layoutemptylinebetweendefs